core/src/apps/cardano/get_address.py

```
@@ -1,48 +1,60 @@
 
     ctx.primary_color, ctx.icon_path = lv.color_hex(PRIMARY_COLOR), ICON
 
     try:
         address = addresses.derive_human_readable(
-            keychain, msg.address_parameters, msg.protocol_magic, msg.network_id
+            keychain, address_parameters, msg.protocol_magic, msg.network_id
         )
     except ValueError as e:
         if __debug__:
             log.exception(__name__, e)
         raise wire.ProcessError("Deriving address failed")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)